spotbugs/src/main/java/edu/umd/cs/findbugs/BugInstance.java

```diff
@@ -1880,7 +1842,7 @@ public String getMessageWithoutPrefix() {
     }
 
     String getLongDescription() {
-        return getBugPattern().getLongDescription().replaceAll("BUG_PATTERN", type);
+        return getBugPattern().getLongDescription().replace("BUG_PATTERN", type);
```

Comment:
The first argument doesn't contain any regex, so we can use `replace()` instead of `replaceAll()`.